docs/src/pages/customization/css-in-js.md

```diff
@@ -90,12 +90,49 @@ This way, we ensure the components always render correctly.
 
 You might, however, also want to override these styles, for example with styled-components.
 If you are experiencing a CSS injection order issue, JSS [provides a mechanism](https://github.com/cssinjs/jss/blob/master/docs/setup.md#specify-dom-insertion-point) to handle this situation.
-By adjusting the placement of the `insertionPoint` comment within your HTML body you can [control the order](http://cssinjs.org/js-api/#attach-style-sheets-in-a-specific-order) that the CSS rules are applied to your components.
+By adjusting the placement of the `insertionPoint` within your HTML head you can [control the order](http://cssinjs.org/js-api/#attach-style-sheets-in-a-specific-order) that the CSS rules are applied to your components.
+
+### HTML comment
+
+The simplest approach is to add an HTML comment that determines where JSS will inject the styles:
+
+```jsx
+<head>
+  <!-- jss-insertion-point -->
+  <link href="..." />
+</head>
+```
+
+```jsx
+import JssProvider from 'react-jss/lib/JssProvider';
+import { create } from 'jss';
+import { createGenerateClassName, jssPreset } from 'material-ui/styles';
+
+const generateClassName = createGenerateClassName();
+const jss = create(jssPreset());
+// We define a custom insertion point that JSS will look for injecting the styles in the DOM.
+jss.options.insertionPoint = 'jss-insertion-point';
+
+function App() {
+  return (
+    <JssProvider jss={jss} generateClassName={generateClassName}>
+      ...
+    </JssProvider>
+  );
+}
+
+export default App;
+```
+
+### HTML noscript
+
+[Create React App](https://github.com/facebookincubator/create-react-app) strips HTML comments when creating the production build.
+To get around the issue, you can use a `<noscript>` element:
```

Comment:
This can be any DOM Node, eg a `noscript` element 

https://github.com/cssinjs/jss/blob/master/docs/setup.md#specify-dom-insertion-point